test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -442,6 +446,31 @@ describe("GET tests", () => {
     expect(resp.text).toContain(CHECK_YOUR_ANSWERS_PAGE_TRUST_TITLE);
   });
 
+  test(`renders the ${CHECK_YOUR_ANSWERS_PAGE} page with trust data and feature flag on`, async () => {
+    mockIsActiveFeature.mockReturnValueOnce(false); // another flag
+    mockIsActiveFeature.mockReturnValueOnce(false); // yet another another flag
```

Comment:
the first 2 feature flags are in the serviceAvailabilityMiddleware. We don't need to test that so we can mock that out. There is then only the one feature flag in the controller get function which you want to test.
You can mock it out in a similar fashion to the authentication middleware (mocked out in this file)
eg

`jest.mock('../../src/middleware/service.availability.middleware');


import { serviceAvailabilityMiddleware } from "../../src/middleware/service.availability.middleware";


const mockServiceAvailabilityMiddleware = serviceAvailabilityMiddleware as jest.Mock;
mockServiceAvailabilityMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );`

Reply:
had a chat with Steve, as I had it working locally I have pushed up the change for this